synapse/models/transport.py

```diff
@@ -52,17 +106,175 @@ def getModelDefs(self):
                 ('transport:sea:telem', ('guid', {}), {
                     'doc': 'A telemetry sample from a vessel in transit.'}),
 
-                # TODO a few more items to plumb eventually
-                # ('transport:sea:hin',
-                # ('transport:sea:port',
+                ('transport:rail:train', ('guid', {}), {
+                    'interfaces': ('transport:trip',),
+                    'template': {
+                        'point': 'gate',
+                        'place': 'station',
+                        'trip': 'train trip',
+                        'vehicle': 'train'},
+                    'doc': 'An individual instance of a consist of train cars running a route.'}),
+
+                ('transport:rail:car', ('guid', {}), {
+                    'interfaces': ('transport:container',),
+                    'template': {'phys:object': 'train car'},
+                    'doc': 'An individual train car.'}),
+
+                ('transport:rail:consist', ('guid', {}), {
+                    'interfaces': ('transport:vehicle',),
+                    'template': {'phys:object': 'train'},
+                    'doc': 'A group of rail cars and locomotives connected together.'}),
+
+                ('transport:shipping:container', ('guid', {}), {
+                    'interfaces': ('transport:container',),
+                    'template': {'phys:object': 'shipping container'},
+                    'doc': 'An individual shipping container.'}),
+
+            ),
+            'interfaces': (
+
+                ('transport:container', {
+                    'interfaces': ('phys:object',),
+                    'doc': 'Properties common to a container used to transport cargo or people.',
+                    'props': (
+
+                        ('built', ('time', {}), {
+                            'doc': 'The date when the {phys:object} was built.'}),
+
+                        ('manufacturer', ('ou:org', {}), {
+                            'doc': 'The organization which manufactured the {phys:object}.'}),
+
+                        ('manufacturer:name', ('ou:name', {}), {
```

Comment:
I realized that for it:prod:soft we have :author -> ps:contact, and for it:prod:hardware and then here, for manufacturer, we are instead using ou:org. Is this something that we will eventually want to make consistent across all forms with a "who/what created this"?

Reply:
Yea, it's probably more correct to make this a `ps:contact` even if we end up tweaking later since it's more flexible. I suppose a vehicle could have been built by an individual. Another use case for the entity:actor abstraction which is coming soon.

Reply:
Maybe, but the lack of the ability to assert and refine later means we'll probably lean the props toward the more common use case. In this instance, production hardware and vehicles are most commonly made by orgs. Changing to a `ps:contact` would change `:manufacturer:name` to a `ps:name` etc which gets icky.